osr2mp4/ImageProcess/Animation/alpha.py

```diff
@@ -1,15 +1,20 @@
-from osr2mp4.ImageProcess import imageproc
 import numpy as np
+from PIL import Image
+
+from osr2mp4.ImageProcess import imageproc
+from osr2mp4.ImageProcess.Animation import easings
 
-def img_fade(img, start, end, step):
+
+
+def img_fade(img: Image, start: float, end: float, step: float):
 	outputs = []
 	for x in np.arange(start, end, step):
 		im = imageproc.newalpha(img, x)
 		outputs.append(im)
 	return outputs
 
 
-def list_fade(img, start, end, step):
+def list_fade(img: [Image], start: float, end: float, step: float):
```

Comment:
I don't think Python type checkers know how to interpret `[Image]`, you're supposed to do `List[Image]` instead (`List` being from `typing`).
(this applied to a bunch of places in the code)

Reply:
aight